### ISSUE TITLE
checking whether package 'Rrd' can be installed ... ERROR

### ISSUE THREAD

**Issue by ebdavison** [pldimitrov/Rrd#3]
When doing the 'R CMD check Rrd' command I get compilation errors:

* Installing *source* package ‘Rrd’ ...
** libs
gcc -I/usr/lib/R/include -g -I/usr/local/include    -fpic  -O3 -g -std=gnu99 -c
Rrd.c -o Rrd.o
Rrd.c: In function ‘importRRD’:
Rrd.c:56: warning: implicit declaration of function ‘access’
Rrd.c:56: error: ‘F_OK’ undeclared (first use in this function)
Rrd.c:56: error: (Each undeclared identifier is reported only once
Rrd.c:56: error: for each function it appears in.)
Rrd.c: At top level:
Rrd.c:178: error: expected ‘)’ before ‘*’ token
Rrd.c: In function ‘smartImportRRD’:
Rrd.c:310: error: ‘rrd_info_t’ undeclared (first use in this function)
Rrd.c:310: error: ‘rrdInfo’ undeclared (first use in this function)
Rrd.c:323: error: ‘F_OK’ undeclared (first use in this function)
Rrd.c:334: warning: implicit declaration of function ‘rrd_info_r’
Rrd.c:343: warning: implicit declaration of function ‘getRraInfo’
Rrd.c:343: warning: assignment makes pointer from integer without a cast
Rrd.c: In function ‘getFirst’:
Rrd.c:498: error: ‘rrd_info_t’ undeclared (first use in this function)
Rrd.c:498: error: ‘rrdInfo’ undeclared (first use in this function)
Rrd.c:510: error: ‘F_OK’ undeclared (first use in this function)
Rrd.c:526: warning: assignment makes pointer from integer without a cast
make: *** [Rrd.o] Error 1
ERROR: compilation failed for package ‘Rrd’
* Removing ‘/usr/src/Rrd.Rcheck/Rrd’

**Comment by andrie** [comment]
I have created a fork of this project at https://github.com/andrie/rrd and will be actively maintaining the project there.  If this is still an issue, please create a new issue at https://github.com/andrie/rrd/issues
